src/Controller/PastriesController.php

```diff
@@ -27,10 +147,244 @@ public function displayPage(Request $request, Response $response, $args)
+        $translator = $this->ci->translator;
+
+        // Load validation rules
+        $schema = new RequestSchema('schema://requests/pastry/create.yaml');
+        $validator = new JqueryValidationAdapter($schema, $translator);
+
+        // Create a dummy pastry to prepopulate fields
+        $pastry = new Pastries();
+
+        $pastry->icon = 'fa fa-cutlery';
+
+        $fields = [
+            'hidden'   => [],
+            'disabled' => []
+        ];
+
+        return $this->ci->view->render($response, 'modals/pastries.html.twig', [
+            'pastry' => $pastry,
+            'form'   => [
+                'action'      => 'api/pastries',
+                'method'      => 'POST',
+                'fields'      => $fields,
+                'submit_text' => 'Create'
+            ],
+            'page' => [
+                'validators' => $validator->rules('json', false)
+            ]
+        ]);
+    }
+
+    public function getModalDelete(Request $request, Response $response, $args)
+    {
+        // GET parameters
+        $params = $request->getQueryParams();
+
+        $pastry = Capsule::table('pastries')->where('name', '=', $params['name'])->first();
+
+        // If the pastry no longer exists, forward to main pastry listing page
+        if (!$pastry) {
+            throw new NotFoundException();
+        }
+
+        /** @var \UserFrosting\Sprinkle\Account\Authorize\AuthorizationManager $authorizer */
+        $authorizer = $this->ci->authorizer;
+
+        /** @var \UserFrosting\Sprinkle\Account\Database\Models\Interfaces\UserInterface $currentUser */
+        $currentUser = $this->ci->currentUser;
+
+        // Access-controlled page
+        if (!$authorizer->checkAccess($currentUser, 'see_pastries')) {
+            throw new ForbiddenException();
+        }
+
+        return $this->ci->view->render($response, 'modals/confirm-delete-pastries.html.twig', [
+            'pastry' => $pastry,
+            'form'   => [
+                'action' => "api/pastries/p/{$pastry->name}",
+            ]
+        ]);
+    }
+
+    public function getModalEdit(Request $request, Response $response, $args)
+    {
+        // GET parameters
+        $params = $request->getQueryParams();
+
+        $pastry = Capsule::table('pastries')->where('name', '=', $params['name'])->first();
+
+        // If the group doesn't exist, return 404
+        if (!$pastry) {
+            throw new NotFoundException();
+        }
+
+        /** @var \UserFrosting\Sprinkle\Account\Authorize\AuthorizationManager $authorizer */
+        $authorizer = $this->ci->authorizer;
+
+        /** @var \UserFrosting\Sprinkle\Account\Database\Models\Interfaces\UserInterface $currentUser */
+        $currentUser = $this->ci->currentUser;
+
+        /** @var \UserFrosting\I18n\MessageTranslator $translator */
+        $translator = $this->ci->translator;
+
+        // Generate form
+        $fields = [
+        'hidden'   => [],
+        'disabled' => []
+    ];
```

Comment:
Careful with indentation here

Reply:
I guess I was relying on php-cs-fixer too much. I had to manually fix the couple weird indentation instances you found.